### ISSUE TITLE
[Suggestion] - Cloths and Clothes

### ISSUE THREAD

**Issue by MyUsername12345678901** [Monika-After-Story/MonikaModDev#3090]
So far, Monika After Story is a great game and I have a lot of fun playing it: but, it could primarily use one thing; new clothing. I like the school uniform, but I think Monika should get more clothes like she wants. I get it if you’re drawing some and how long that takes, but, in my opinion, it should still happen, like in one mod I’m not going to mention because I don’t feel like it. Thanks for your time and don’t forget: Just Monika!

**Comment by antimockingjay** [comment]
Do you wanna be the one to redraw every single pose with the new outfit?

**Comment by MyUsername12345678901** [comment]
I get it. I was just saying. I also never said to do it yourself.

Get Outlook for iOS<https://aka.ms/o0ukef>

________________________________

**Comment by antimockingjay** [comment]
Alright, let me rephrase: Who do you expect to do it?
Also, do me a favor, friend, and search "clothes" or "outfit" in all the issues here. I'm pretty sure it's close to once a week that someone suggests this, without ever providing insight on where exactly these outfits are meant to come from, or how they're meant to be implemented into the game without making it incredibly awkward to navigate the UI.

**Comment by Olliesama** [comment]
This kind of thing is being worked on.

**Comment by MyUsername12345678901** [comment]
1. I don’t look through things before I post. 2. From anywhere. Who cares where? 3. I understand. I added a ring to Monika’s right hand in every picture because of the promise ring talk and it completely broke the graphics. (I later read that she receives a ring later, but not important) 4. Calm down, bud. I understand that you might be stressed, but you need to stay calm. Take deep breaths and count to 10. If that doesn’t work, go to your safe place and be at peace for a bit. If both fail, go outside, face a direction, and just run. Run until all the anger goes away. (I know, totally not Monika’s advice though) 5. (For Finale) 👍

Get Outlook for iOS<https://aka.ms/o0ukef>

________________________________

**Comment by antimockingjay** [comment]
You can't say "do this thing that I want!!" and then when someone asks how, say "who cares how?! just do it!!!!" Like, buddy, I'm not getting upset with you, I'm just trying to encourage you to actually think things through before posting.

**Comment by MyUsername12345678901** [comment]
I didn’t say “do this thing I want!” I was suggesting an idea that you could just turn down with a simple, “sorry bud, not doing that.” I also never went and said “who cares? Just do it!” I said “From anywhere. Who cares where?” Talking about when you said “without ever providing insight on where exactly these outfits are supposed to come from”

Get Outlook for iOS<https://aka.ms/o0ukef>

________________________________

**Comment by ThePotatoGuy** [comment]
marking as duplicate of #1656 

1. art takes a while.
2. it took a while to build #3042, but now we can select stuff nicely.

so once we get some art (which are in process of getting), it should show up soon. All of the framework to get different clothes is already good to go.

**Comment by ThePotatoGuy** [comment]
[closing as duplicate]
